jdaviz/configs/default/plugins/model_fitting/tests/test_plugin.py

```diff
@@ -385,3 +385,56 @@ def test_invalid_subset(specviz_helper, spectrum1d):
 
     plugin.dataset = 'left_spectrum'
     assert plugin._obj.spectral_subset_valid
+
+
+def test_all_nan_uncert(specviz_helper):
+
+    # test that if you have a fully finite data array, and a fully nan/inf
+    # uncert array, that it is set to None and the fit proceeds (rather than
+    # being filtered in the fitter, as would normally happen with nans)
+
+    uncertainty = StdDevUncertainty([np.nan, np.nan, np.nan, np.nan, np.nan, np.nan] * u.Jy)
```

Comment:
Does not really matter here since you only have 6 elements, though you could also have used https://numpy.org/doc/stable/reference/generated/numpy.full.html

Reply:
I thought it made it easier to see visually what elements are being compared here in the test, for when i come back to this in the future and forgot what the issue was